routes/api.php

```diff
@@ -26,3 +27,8 @@
 Route::middleware(['auth:sanctum', 'ensureAdmin'])->group(function () {
     Route::apiResource('/users', UserController::class);
 });
+
+Route::middleware(['auth:sanctum'])->group(function () {
+    Route::apiResource('/tasks', TaskController::class);
```

Comment:
```suggestion
    Route::apiResource('/tasks', TaskController::class, , ['only' => 'show']);
```

とすると、showだけがルーティングされると思うのでトライしてみてください。